packages/palette/src/svgs/Icon.tsx

```diff
@@ -44,3 +44,27 @@ Icon.defaultProps = {
   height: "18px",
   width: "18px",
 }
+/** Compatibility component used to normalize paths between react dom and react native */
+export const Path: FC<JSX.IntrinsicElements["path"]> = ({ ...props }) => {
+  return <path {...props} />
+}
+
+/** Compatibility component used to normalize titles between react dom and react native */
+export const Title: FC<JSX.IntrinsicElements["title"]> = ({ ...props }) => {
+  return <title {...props} />
+}
+
+/** Compatibility component used to normalize svg groups between react dom and react native */
+export const G: FC<JSX.IntrinsicElements["g"]> = ({ ...props }) => {
+  return <g {...props} />
+}
+
+/** Compatibility component used to normalize svg circles between react dom and react native */
+export const Circle: FC<JSX.IntrinsicElements["circle"]> = ({ ...props }) => {
+  return <circle {...props} />
+}
+
+/** Compatibility component used to normalize svg rects between react dom and react native */
+export const Rect: FC<JSX.IntrinsicElements["rect"]> = ({ ...props }) => {
+  return <rect {...props} />
+}
```

Comment:
So I guess there's some magic here? This set of components makes the icons cross-platform compatible? Is it that `IntrinsicElements` thingy?